lib/util/cache/package/ttl.ts

```diff
@@ -0,0 +1,43 @@
+import is from '@sindresorhus/is';
+import { GlobalConfig } from '../../../config/global';
+import type { PackageCacheNamespace } from './types';
+
+export function getTtlOverride(
+  namespace: PackageCacheNamespace,
+): number | undefined {
+  const ttl: unknown = GlobalConfig.get('cacheTtlOverride', {})[namespace];
```

Comment:
```suggestion
  const ttl = GlobalConfig.get('cacheTtlOverride')?.[namespace];
```
is the unknown type def really needed?